libs/common-services/src/main/java/uk/gov/di/ipv/core/library/dto/CriCallbackRequest.java

```diff
@@ -15,7 +17,10 @@
 @Builder
 public class CriCallbackRequest {
     private String authorizationCode;
-    private String credentialIssuerId;
+
+    @JsonProperty("credentialIssuerId")
+    private Cri credentialIssuer;
```

Comment:
(I don't _love_ having our model classes different to the JSON structure, but I can appreciate the convenience)

Reply:
same, I can change back to a string then provide an extra method, getCri to parse it into the enum?

Reply:
Yeah that might be cleaner - and saves you the serialisation classes (and you can `@JsonIgnore` it)